source/nodejs/adaptivecards-site/pages/_data/home.yml

```diff
@@ -91,15 +91,19 @@ en:
     - title: Native performance
       subtitle: Adaptive Cards render native UI on any platform
       smallImage: content/Feature.jpg
+      smallImageAlt: Native performance
     - title: Adapts to the surrounding UI
       subtitle: Cards automatically adaptive to the surrounding UI
       smallImage: content/Feature.jpg
+      smallImageAlt: Adapts to the surrounding UI
     - title: Fully extensible
       subtitle: The schema is open-ended so you can add your own element
       smallImage: content/Feature.jpg
+      smallImageAlt: Fully extensible
     - title: Dynamic and interactive
       subtitle: Interactivity is expressed declaratively to help reduce risk of custom code injection
       smallImage: content/Feature.jpg
+      smallImageAlt: Dynamic and interactive
```

Comment:
All these images are the same, and to be honest I'm not sure what the correct alt text should be so I just used the titles for the alt texts

Reply:
Since these are decorative images, I suggest doing what this part of [WCAG 2.1](https://www.w3.org/TR/2018/REC-WCAG21-20180605/):
> Decoration, Formatting, Invisible
: If non-text content is [pure decoration](https://www.w3.org/TR/2018/REC-WCAG21-20180605/#dfn-pure-decoration), is used only for visual formatting, or is not presented to users, then it is implemented in a way that it can be ignored by [assistive technology](https://www.w3.org/TR/2018/REC-WCAG21-20180605/#dfn-assistive-technologies).

Reply:
Good call! PR has been updated to just have these images have `role="none"` so they don't show up to accessibility insights